### ISSUE TITLE
How make winfetch use the config

### ISSUE THREAD

**Issue by noxz-dev** [M4cs/winfetch#5]
Hi,

how do I get it that winfetch uses my configuration instead of the built-in ?

thx

nox

Edit: got it to work :)

still got a problem:

its placing the asciiart under the informations, how to fix this ?

**Comment by Lyapsus** [comment]
So how do I do that?

**Comment by M4cs** [comment]
@HeyImNoxz Are you stilll having that issue? Sorry I didn't get any notifications for this issue thread

**Comment by noxz-dev** [comment]
Yeah but the start of the console took to long so im not using it anymore
